src/lib/hosts-file.ts

```diff
@@ -4,6 +4,7 @@ import { platform, tmpdir } from 'os';
 import path from 'path';
 import { promisify } from 'util';
 import * as Sentry from '@sentry/electron/main';
+import escapeRegExp from 'lodash/escapeRegExp';
```

Comment:
Studio uses Electron 35, which uses Chrome 134. There is `RegExp.escape()` added in Chrome 136 so we can move to this one when we upgrade Electron.